File: token-validation-jaxrs/src/test/java/no/nav/security/token/support/jaxrs/rest/WithoutAnnotationsResource.java
package no.nav.security.token.support.jaxrs.rest;

import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;
import jakarta.ws.rs.core.Response;

@Path("without/annotations")
public class WithoutAnnotationsResource {

    @GET
    public Response get() {
        return Response.ok().build();
    }

}

File: token-validation-jaxrs/src/test/kotlin/no/nav/security/token/support/jaxrs/rest/WithoutAnnotationsResource.kt
package no.nav.security.token.support.jaxrs.rest

import jakarta.ws.rs.GET
import jakarta.ws.rs.Path
import jakarta.ws.rs.core.Response.ok

@Path("without/annotations")
class WithoutAnnotationsResource {

    @GET
    fun get() = ok().build()
}